Tests/UI/New UI/BillingForm/View/ExpiryDateViewTests.swift

```diff
@@ -89,4 +89,139 @@ class ExpiryDateViewTests: XCTestCase {
         view.updateExpiryDate(to: "Test")
         XCTAssertFalse(view.style?.error?.isHidden ?? true)
     }
+
+  func testValidFirstDigitInputWith0() {
```

Comment:
Can you please also add few test scenarios with garbage or invalid values and strings : for example ```
textField.text = "adbadfa", textField.text = "99/99" textField.text = "00/00" textField.text = "999999999/999999999" etc 
textField.text = "-*/@@"
```

Reply:
Update